frontend/packages/metal3-plugin/src/components/dashboard/inventory-card.tsx

```diff
@@ -64,7 +64,7 @@ export const InventoryCard: React.FC<InventoryCardProps> = ({
 
   const podResult = prometheusResults.getIn([queries[HostQuery.NUMBER_OF_PODS], 'result']);
   const podError = getHostQueryResultError(podResult);
-  const podCount = _.get(podResult, 'data.result', []).length;
+  const podCount = _.get(podResult, 'data.result', [{value: [0, 0]}])[0].value[1];
```

Comment:
I think it would be better to parse the result with [getInstantVectorStats](https://github.com/openshift/console/blob/master/frontend/public/components/graphs/utils.ts#L14)

Reply:
`getInstantVectorStats` unnecessarily parses all the values in the list; we only need the first one. And if the `data.result` list is empty, it'll return an empty list, and we still have to check for that, and then extract the value.

```js
const podCountValues = getInstantVectorStats(podResult);
let podCount = 0;
if (podCountValues.length) {
  podCount = podCountValues[0].y;
}
```

Is that really better?

Reply:
there is always just one value in the query (since you dont specify time range).

I still think its more readable to use `getInstantVectorStats` as `[{value: [0, 0]}])[0].value[1]` looks like a magic to me :)

Im also not sure if showing 0 when there is not result is ok - unavailable would be better IMO.